<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBugData</class>
 <widget class="QWidget" name="NewBugData">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="EnvLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="BugNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="DescriptionLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="AffectedSystemLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="AffectedProdLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>EnvLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EnvLineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
